src/hooks/UseRequirementData/UseRequirementData.js

```diff
@@ -45,125 +46,181 @@ const UseRequirementData = (files) => {
     };
     
     // post given file
-    // const postFile = async (file, index) => {
-    //     try {
-    //         const base64Content = await fileToBase64(file);
-    //         const response = await axios.post(createURL(), {
+    const postFile = async (file, index) => {
+        try {
+            const base64Content = await fileToBase64(file);
+            const response = await axios.post(createURL(), {
+                mode: 'no-cors',
```

Comment:
You might need to remove this because the website will need to use cors when testing it by deploying the stack with new front-end changes